app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/ReturnController.scala

```diff
@@ -42,26 +40,13 @@ import scala.util.Try
 class ReturnController @Inject() (cc: ControllerComponents, appConfig: AppConfig)
     extends BackendController(cc)
     with Logging {
-  private lazy val schemaToBeValidated = Json
-    .fromJson[SchemaType](
-      Json.parse(
-        Source
-          .fromInputStream(
-            this.getClass.getResourceAsStream("/resources/submit-return-des-schema-v-1-0-0.json")
-          )
-          .mkString
-      )
-    )
-    .get
 
   def submitReturn(cgtReferenceNumber: String): Action[JsValue] =
     Action(parse.json) { request =>
-      val validator = SchemaValidator(Some(Version4))
-
       val submittedReturn: JsResult[(BigDecimal, LocalDate, JsValue)] = for {
         a <- (request.body \ "ppdReturnDetails" \ "returnDetails" \ "totalLiability").validate[BigDecimal]
```

Comment:
it would be nice to replace this block with something like

`json.validate[Return]`